apps/spruce/src/pages/version/Tabs/VersionTiming/VersionTimingGraph/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { palette } from "@leafygreen-ui/palette";
+import { ListSkeleton } from "@leafygreen-ui/skeleton-loader";
+import Chart from "react-google-charts";
+import { useNavigate } from "react-router-dom";
+import { getTaskRoute, getVersionRoute } from "constants/routes";
+import { VersionPageTabs } from "types/patch";
+import { applyStrictRegex } from "utils/string";
+import { GanttChartData } from "../types";
+
+const { black, green } = palette;
+const CHART_ROW_HEIGHT_IN_PIXELS = 42;
+const MINIMUM_CHART_HEIGHT_IN_PIXELS = 400;
+const DEFAULT_LABEL_MAX_WIDTH = 300;
+const CHARACTER_WIDTH = 8;
+
+interface Props {
+  versionId: string;
+  data: GanttChartData;
+  loading: boolean;
+  dataType: "task" | "variant";
+}
+
+const VersionTimingGraph: React.FC<Props> = ({
+  data,
+  dataType,
+  loading,
+  versionId,
+}) => {
+  const navigate = useNavigate();
+
+  if (loading) {
+    return <ListSkeleton />;
+  }
+
+  if (!data || data.length === 1) {
+    return (
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: `${MINIMUM_CHART_HEIGHT_IN_PIXELS}px`,
+        }}
+      >
+        <p>No data available</p>
+      </div>
```

Comment:
Can we use emotion to style this element? 

it would look something like this.

Declare the styled component. 
```ts
const Container = styled.div`
display: flex;
justify-content: center;
align-items: center;
height: ${MINIMUM_CHART_HEIGHT_IN_PIXELS}px;
`
```

And then you can just reuse the component .
```suggestion
      <Container>
              <p>No data available</p>
      </Container>

```